lib/oper1.g

```diff
@@ -745,7 +745,7 @@ InstallAttributeFunction(
 ##  <#/GAPDoc>
 ##
 IsPrimeInt := "2b defined";
-
+MakeWriteOnceAtomic := "2b defined";
```

Comment:
`MakeWriteOnceAtomic()` is a kernel function, so the only reason to have it there is for when oper1.g is getting precompiled (maybe not even then?). Otherwise, it should already exist and it seems like it would be overwritten. Or am I missing something here?